### DISCUSSION TITLE
Add a toggle to patches/server/0075-Use-a-Shared-Random-for-Entities.patch

### DISCUSSION THREAD

**Discussion by obaba-samich** [PaperMC/Paper#11616] [Ideas]
### Is your feature request related to a problem?

 I don't want this to be enabled on my server because me and my friend suck at Minecraft 😅

### Describe the solution you'd like.

Just add a toggle or tell me how to do it _and read the other part below_ because I can't read patches nor do I know how to build paper from source.

### Describe alternatives you've considered.

Reading Purpur's patch, but I am incapable.

### Other

Located [here](https://github.com/PurpurMC/Purpur/blob/ver/1.21.3/patches/server/0248-Add-toggle-for-RNG-manipulation.patch), this is the patch I'm talking about (purpur), so if you could, help me build this instead?

**Comment by ryantheleach** [comment]
I'm generally in support of being able to vanilla-ize Paper using configuration, but understand that additional configuration options adds complexity of understanding to server admins, so picking and choosing what patches have config options is worthwhile.

As far as I know, the only thing this would change @PizzaGod873 , is breaking vanilla compatibility with stuff like villager trades for set-seed speedruns, in which case I have to wonder at the wisdom of using Paper in the first place.

But maybe there's some edge-case somewhere where someone would want this...
